static/app/views/replays/deadRageClick/deadRageSelectorCards.tsx

```diff
@@ -1,97 +1,264 @@
+import TextOverflow from 'sentry/components/textOverflow';
 import {IconCursorArrow} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import useDeadRageSelectors from 'sentry/utils/replays/hooks/useDeadRageSelectors';
 import {useLocation} from 'sentry/utils/useLocation';
-import SelectorTable from 'sentry/views/replays/deadRageClick/selectorTable';
+import useOrganization from 'sentry/utils/useOrganization';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+import Accordion from 'sentry/views/performance/landing/widgets/components/accordion';
+import {RightAlignedCell} from 'sentry/views/performance/landing/widgets/components/selectableList';
+import {
+  ContentContainer,
+  HeaderContainer,
+  HeaderTitleLegend,
+  StatusContainer,
+  Subtitle,
+  WidgetContainer,
+} from 'sentry/views/profiling/landing/styles';
+import ExampleReplaysList from 'sentry/views/replays/deadRageClick/exampleReplaysList';
 
 function DeadRageSelectorCards() {
-  const location = useLocation();
-
   return (
     <SplitCardContainer>
-      <DeadClickTable location={location} />
-      <RageClickTable location={location} />
+      <AccordionWidget
+        clickType="count_dead_clicks"
+        widgetTitle={t('Most Dead Clicks')}
+        deadOrRage="dead"
+        tooltip={t('The top selectors your users have dead clicked on.')}
+      />
+      <AccordionWidget
+        clickType="count_rage_clicks"
+        widgetTitle={t('Most Rage Clicks')}
+        deadOrRage="rage"
+        tooltip={t('The top selectors your users have rage clicked on.')}
+      />
     </SplitCardContainer>
   );
 }
 
-function DeadClickTable({location}: {location: Location<any>}) {
+function AccordionWidget({
+  clickType,
+  deadOrRage,
+  widgetTitle,
+  tooltip,
+}: {
+  clickType: 'count_dead_clicks' | 'count_rage_clicks';
+  deadOrRage: 'dead' | 'rage';
+  tooltip: string;
+  widgetTitle: string;
+}) {
+  const [selectedListIndex, setSelectListIndex] = useState(0);
   const {isLoading, isError, data} = useDeadRageSelectors({
-    per_page: 4,
-    sort: '-count_dead_clicks',
+    per_page: 3,
+    sort: `-${clickType}`,
     cursor: undefined,
     prefix: 'selector_',
     isWidgetData: true,
   });
+  const location = useLocation();
+  const filteredData = data.filter(d => (d[clickType] ?? 0) > 0);
 
   return (
-    <SelectorTable
-      data={data.filter(d => (d.count_dead_clicks ?? 0) > 0)}
-      isError={isError}
-      isLoading={isLoading}
-      location={location}
-      clickCountColumns={[{key: 'count_dead_clicks', name: 'dead clicks'}]}
-      title={
-        <Fragment>
-          <IconContainer>
-            <IconCursorArrow size="xs" color="yellow300" />
-          </IconContainer>
-          {t('Most Dead Clicks')}
-        </Fragment>
-      }
-      customHandleResize={() => {}}
-      clickCountSortable={false}
-    />
+    <WidgetContainer>
```

Comment:
We should remove the bottom margin from WidgetContainer. LayoutGap does the same job.